packages/web-app-files/src/views/Personal.vue

```diff
@@ -225,17 +295,20 @@ export default {
 
       visibilityObserver.observe(component.$el, { onEnter: debounced, onExit: debounced.cancel })
     },
-
+    async getFolderItems(path, properties) {
+      try {
+        const resources = await this.$client.files.list(path, 1, properties)
+        return resources
+      } catch (error) {
+        console.error(error)
+      }
+    },
     async loadResources(sameRoute) {
       this.loading = true
       this.CLEAR_CURRENT_FILES_LIST()
 
       try {
-        let resources = await this.$client.files.list(
-          this.$route.params.item,
-          1,
-          DavProperties.Default
-        )
+        let resources = await this.getFolderItems(this.$route.params.item, this.davProperties)
```

Comment:
Benedikt introduced `DavProperties.Default` constants, you've switched back to `this.davProperties` for a reason?